src/vfd/sriov.c

```diff
@@ -356,9 +356,11 @@ tx_set_loopback(portid_t port_id, u_int8_t on)
 */
 int get_split_ctlreg( portid_t port_id, uint16_t vf_id ) {
 #ifdef BNXT_SUPPORT
-	bleat_printf( 0, "NOT reading split ctlreg on port=%d vf=%d", port_id, vf_id );
-	return 0;
-#else
+	if (strcmp(rte_eth_devices[port_id].driver->pci_drv.driver.name, "net_bnxt") == 0) {
```

Comment:
If we are going to check the driver_name, then the ifdef BNXT_SUPPORT is redundant.